google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudApiManager.java

```diff
@@ -430,7 +431,8 @@ private static void notifyApiEnableSkipped(Set<CloudLibrary> apis, Project proje
     notification.notify(project);
   }
 
-  private static void notifyServiceAccountCreated(Project project, String name, Path downloadDir) {
+  private static void notifyServiceAccountCreated(
+      Project project, String gcpProjectId, String name, Path downloadDir) {
```

Comment:
Could we possibly pass `CloudProject` into both this method and `ServiceAccountKeyDisplayDialog` to make it a bit more strong typed and easier to see that we get project ID from it?